src/commands/setup-for-local-development.js

```diff
@@ -0,0 +1,122 @@
+const Listr = require('listr');
+
+const { flags: flagTypes } = require('@oclif/command');
+
+const BaseCommand = require('../oclif/command/BaseCommand');
+const UpdateRendererWithOutput = require('../oclif/renderer/UpdateRendererWithOutput');
+const MuteOneLineError = require('../oclif/errors/MuteOneLineError');
+
+const PRESETS = require('../presets');
+
+class SetupForLocalDevelopmentCommand extends BaseCommand {
+  /**
+   *
+   * @param {Object} args
+   * @param {Object} flags
+   * @param {generateToAddressTask} generateToAddressTask
+   * @param {registerMasternodeTask} registerMasternodeTask
+   * @param {initTask} initTask
+   * @param {DockerCompose} dockerCompose
+   * @return {Promise<void>}
+   */
+  async runWithDependencies(
+    { port: coreP2pPort, 'external-ip': externalIp },
+    {
+      'drive-image-build-path': driveImageBuildPath,
+      'dapi-image-build-path': dapiImageBuildPath,
+    },
+    generateToAddressTask,
+    registerMasternodeTask,
+    initTask,
+    dockerCompose,
+  ) {
+    const preset = PRESETS.LOCAL;
+    const network = 'testnet';
+    const amount = 2000;
+
+    const tasks = new Listr([{
+      title: 'Setup masternode for local development',
+      task: () => (
+        new Listr([
+          {
+            title: `Generate ${amount} dash to address`,
+            task: () => (
+              generateToAddressTask(amount)
+            ),
+          },
+          {
+            title: 'Wait core to stop',
+            task: async () => {
+              const stopAllContainers = this.container.resolve('stopAllContainers');
+              const startedContainers = this.container.resolve('startedContainers');
+
+              await stopAllContainers(startedContainers.getContainers());
+            },
+          },
+          {
+            title: 'Register masternode',
+            task: () => (
+              registerMasternodeTask()
+            ),
+          },
+          {
+            title: 'Wait core to stop',
+            task: async () => {
+              const stopAllContainers = this.container.resolve('stopAllContainers');
+              const startedContainers = this.container.resolve('startedContainers');
+
+              await stopAllContainers(startedContainers.getContainers());
+            },
+          },
+          {
+            title: 'Initialize platform',
+            task: () => (
+              initTask(
+                preset,
+              )
+            ),
+          },
+        ])
+      ),
+    },
+    ],
+    { collapse: false, renderer: UpdateRendererWithOutput });
+
+    try {
+      await tasks.run({
+        preset,
+        externalIp,
+        coreP2pPort,
+        network,
+        driveImageBuildPath,
+        dapiImageBuildPath,
+      });
+    } catch (e) {
+      throw new MuteOneLineError(e);
+    } finally {
+      await dockerCompose.down(preset);
```

Comment:
Why do we need it here? We have https://github.com/dashevo/mn-bootstrap/blob/245770e08400168b61a583a0f1dae695c41bd277/src/oclif/command/BaseCommand.js#L49

`initTask` supposed to stop everything right?

Reply:
We have to stop everything in case of some error happened